resources/views/layouts/ixpv4.foil.php

```diff
@@ -52,14 +52,14 @@
             <div class="row" >
 
                 <?php if( Auth::check() && Auth::user()->isSuperUser()
-                        && !Auth::user()->is2faRequired() && Auth::user()->is2faAuthenticated()
+                        && !Auth::user()->is2faRequired() && !Auth::user()->is2faAuthRequiredForSession()
```

Comment:
Re: `&& !Auth::user()->is2faRequired()`

Can yiou incorporate this check into is2faAuthRequiredForSession()?